docs/src/views/Theming.tsx

```diff
@@ -97,12 +109,57 @@ export default () => (
       the <NavLink to="components/provider">Provider</NavLink>.
     </p>
     <ExampleSnippet
+      value={`
+        <>
+          <Icon name="user" circular />
+          <Icon name="user" circular />
+
+          <Provider
+            theme={{
+              componentVariables: {
+                Icon: {
+                  iconColor: 'cornflowerblue'
+                  }
+              },
+              componentStyles: {
```

Comment:
again, while this fixes an example, this makes it being non-aligned with the docs section: here, essentially, we are demoing _styles_ customization (even if based on _variables_ - but this complicates example even more). Lets agree on the resolution for this comment https://github.com/stardust-ui/react/pull/1625/files#r305273759, and then we would be able to address this issue in the same way.